platform-modules/mimetype-manager/src/main/scala/org/sunbird/models/UploadParams.scala

```diff
@@ -0,0 +1,26 @@
+package org.sunbird.models
+
+import play.api.mvc.QueryStringBindable
+
+
+case class UploadParams(fileFormat: String = "", validation: Boolean = true)
+
+object UploadParams {
```

Comment:
Move this object to the content-service module.

Reply:
removed query string binder